debezium-core/src/main/java/io/debezium/pipeline/metrics/ChangeEventSourceMetricsMXBean.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.pipeline.metrics;
+
+/**
+ * Metrics that are common for both snapshot and streaming change event sources
+ *
+ * @author Jiri Pechanec
+ *
+ */
+public interface ChangeEventSourceMetricsMXBean {
+
+    String getLastEvent();
+    long getSecondsSinceLastEvent();
```

Comment:
Seconds seems a bit too coarse, as there can (and hopefully are) multiple events per second. How about changing it to milli-seconds at least?